### DISCUSSION TITLE
Some admonitions have bad color scheme in dark theme

### DISCUSSION THREAD

**Discussion by astrojuanlu** [pradyunsg/furo#122] [Potential Issue] [answered]
Just spotted this: https://diataxis.fr/

![Screenshot_2021-04-03 Home - Diátaxis](https://user-images.githubusercontent.com/316517/113488480-b79aba80-94be-11eb-830b-2f9850b62266.png)

![Screenshot from 2021-04-03 20-54-22](https://user-images.githubusercontent.com/316517/113488491-c8e3c700-94be-11eb-8851-80159b123079.png)

**Comment by pradyunsg** [comment, marked as answer]
That's something custom done by that documentation; I'd suggest taking this up with them.
